pywt/tests/test_mra.py

```diff
@@ -17,6 +18,7 @@
 # 1d mra tests
 ####
 
+@uses_futures
```

Comment:
This doesn't use `futures`, right? Looks like the changes in this file either are a left-over that need removing, or otherwise needs a `@pytest.mark.skipif(IS_WASM, reason="...")`?

Reply:
It doesn't use `futures`, yes – addressed in https://github.com/PyWavelets/pywt/pull/701/commits/4a962c9e78e0f581c777b79734e9592d973ba918.

I had integrated both the `IS_WASM` and the `ImportError` checks under the `uses_futures` condition that's currently used under a marker.